### ISSUE TITLE
Window Content Scale not used (instead used from project settings)

### ISSUE THREAD

**Issue by Gamemap** [godotengine/godot#68309]
### Godot version

v4.0.beta3.official [01ae26d31]

### System information

Windows 11, Vulkan

### Issue description

I am trying to make a second window with a diffrent scale than the main window (with its settings in Project Settings) but the anchors of the control node scale the content outside the window. (You can't see the label `test1` and neither the red image in the minimal reproduction project.) I set the `Size` and the `Content Size` of the `Window` node the same like the root node in the scene. 

I think the second window uses the same size for the anchors as the main window and not from its node setting.
(Please tell me if this is wrong)

### Steps to reproduce

In the control.tscn scene you see the size of the window with all content on green background. On the left side (red color on top) has a `Tree` node with `Expanding` enabeled to see the problem with the right side. If you run the program only the normal godot logo (inside the blue boundary) is visible and the red godot logo is outside the window.
All nodes have the anchors set to current ratio and the size of 400x600 (the 2 images are smaller). The window node has the Viewport Mode and the Expand aspect.

### Minimal reproduction project

[WindowContentScaleBug.zip](https://github.com/godotengine/godot/files/9944975/WindowContentScaleBug.zip)

**Comment by Calinou** [comment]
Can you reproduce this after disabling the **Embed Subwindows** project setting? By default, subwindows are embedded when running the project.

**Comment by Gamemap** [comment]
Yes, it also happens with disabled embed subwindows. (It should be already disabled in the minimal reproduction project.)

**Comment by Sauermann** [comment]
In the config of your `$"/root/main/Window/Control"` you have as bottom anchor point the value `3`.
This `3` means that it should only display the top third of its area.

![image](https://user-images.githubusercontent.com/6299227/200186568-77b9066c-5a33-40f6-a4e3-01a12349721d.png)

Changing this value to `1` should resolve your issue for the running app.

This change has the effect, that the scene looks wrong in the editor-display (stretched vertivally). This is based on the fact, that there is only a single viewport-size setting in the project settings for all scenes.
Maybe this could be resolved by having a separate viewport-size setting for every scene?
